ccd-definition/CaseField.json

```diff
@@ -542,6 +542,13 @@
     "FieldType": "OrganisationPolicy",
     "SecurityClassification": "Public"
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "applicantSolicitor1ServiceAddress",
```

Comment:
we need new fields in CaseData model in civil-service repo before merging this PR

Reply:
got civil service  PR ready